frontend/src/properties/propertysections/TextfieldPropertySection.tsx

```diff
@@ -80,6 +93,15 @@ export const TextfieldPropertySection = ({
   subscribers,
   readOnly,
 }: TextfieldPropertySectionProps) => {
+  let bgColor = 'white';
+  let fgColor = 'black';
+  if (widget.style) {
+    bgColor = widget.style.backgroundColor;
+    fgColor = widget.style.foregroundColor;
+  }
+  const props: StyleProps = { backgroundColor: bgColor, foregroundColor: fgColor };
```

Comment:
You don't really need those default values since they can't be used anywhere. Something like this would not make someone believe that the widget can be black and white:

```
const props: StyleProps = {
  backgroundColor: widget.style?.backgroundColor ?? null,
  foregroundColor: widget.style?.foregroundColor ?? null,
};
```

This way both in the TypeScript types and the code, everything indicates that both the style and the individual properties can be `null`. The check on `backgroundColor ?` in the useStyle above and the check on `widget.style` both make more sense.

Reply:
Done